src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for the 'side' and pass it on for better code readability.